### PR TITLE
feat(dev tools): Add initial dev tools.

### LINKED ISSUE #89: Add devtools panel
Enable:
* Show currently available card selection
* Show currently available events

### PR DESCRIPTION
**Pull request by Greenheart**
Note: formatting of existing code occured, but is should be kept to enforce code style.

Fix #89

### CODE REVIEW

File: src/components/Game.js [hunk 1]
```diff
@@ -110,6 +114,8 @@ export default class Game extends Component {
     getNextCard(updatedWorld, card, currentAction) {
         const { eventCards } = this.props.worldData
         const availableEvents = this.getAvailableEvents(updatedWorld)
+        let availableCards = []
```
Comment (Muthaias) [inline src/components/Game.js:117]: Why this W1?

File: src/components/Game.js [hunk 2]
```diff
@@ -130,10 +136,18 @@ export default class Game extends Component {
         } else if (eventStartingNow) {
             nextCard = this.selectEventCard(eventStartingNow.initialEventCardId)
         } else {
-            const availableCards = this.getAvailableCards(updatedWorld)
```
Comment (Muthaias) [inline src/components/Game.js:133]: Why this W1?

File: src/components/Game.js (hunk 1 above)
Comment (Muthaias) [reply src/components/Game.js:117]: To me it seems like availableCards should be a const atleast?

**Comment by Greenheart** [comment]
Resolved after discussion

**Pull request merged**